src/participant_network.star

```diff
@@ -88,15 +81,36 @@ def launch_participant_network(
         batcher_params,
     )
 
-    # The OP Stack don't run the proposer anymore, it has been replaced with the challenger
-    # op_proposer_launcher.launch(
-    #     plan,
-    #     "op-proposer{0}".format(l2_services_suffix),
-    #     input_parser.DEFAULT_PROPOSER_IMAGES["op-proposer"],
-    #     all_cl_contexts[0],
-    #     l1_config_env_vars,
-    #     gs_private_keys["GS_PROPOSER_PRIVATE_KEY"],
-    #     l2oo_address,
-    # )
+    game_factory_address = util.read_network_config_value(
+        plan,
+        deployment_output,
+        "state",
+        ".opChainDeployments[0].disputeGameFactoryProxyAddress",
+        # ".opChainDeployments[{0}].disputeGameFactoryProxyAddress".format(
+        #     1
+        # ),
```

Comment:
just noticed this, will fix so it reads the correct address in a multi-network setup